airflow-ctl/src/airflowctl/api/operations.py

```diff
@@ -554,11 +554,10 @@ def trigger(
         self, dag_id: str, trigger_dag_run: TriggerDAGRunPostBody
     ) -> DAGRunResponse | ServerResponseError:
         """Create a dag run."""
+        if trigger_dag_run.conf is None:
+            trigger_dag_run.conf = {}
```

Comment:
Instead of this is it not easier to define `{}` as default in the `TriggerDAGRunPostBody`?

Reply:
I double-checked and see that the underlying core methods allow empty `conf`. Since dict is in the factory, I think allowing None will resolve as `{}` in the route method

https://github.com/apache/airflow/blob/c2059b9394d021cf955cf8bae13da8c39965bd92/airflow-core/src/airflow/models/dag.py#L1511

Reply:
Updated as it can be `None` means factory will end up as `{}`, it worked in Postman too.